nautobot/extras/views.py

```diff
@@ -903,7 +903,9 @@ def check_and_call_git_repository_function(request, pk, func):
 
     # Allow execution only if a worker process is running.
     if not get_worker_count():
-        messages.error(request, "Unable to run job: Celery worker process not running.")
+        msg = "Unable to run job: Celery worker process not running."
+        messages.error(request, msg)
+        return HttpResponseBadRequest(msg)
```

Comment:
It might be a nicer user experience if we instead redirected back to the original view (GitRepositorySyncView/GitRepositoryDryRunView) rather than displaying a 400 error page?

Reply:
I agree I think that is better given the flash capabilities of the Django messages. But I'll be real honest, I don't know how to do it correctly. Taking a guess in my most recent push.